### PR TITLE
Use `yarn --cwd` parameter instead of `cd` pattern for exampls in documentation

### PR DESCRIPTION
**Pull request by sisidra**
## Hey, I just made a Pull Request!

I think it is considered a good practice to use --cwd parameter and staying in project root, instead of changing working directory.
If this is not the case, we can close PR... :)

#### :heavy_check_mark: Checklist

- [ ] ~A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))~
- [x] Added or updated documentation
- [ ] ~Tests for new functionality and regression tests for bug fixes~
- [ ] ~Screenshots attached (for UI changes)~

### CODE REVIEW

File: docs/features/techdocs/getting-started.md [hunk 1]
```diff
@@ -62,8 +61,7 @@ Navigate to `packages/backend` of your Backstage app, and install the
 
 ```bash
 cd my-backstage-app/
-cd packages/backend
-yarn add @backstage/plugin-techdocs-backend
+yarn --cwd packages/backend add @backstage/plugin-techdocs-backend
```
Comment (OrkoHunter) [inline docs/features/techdocs/getting-started.md:64]: I kind of like breaking this into two smaller, simpler lines and being more explicit. :)
But I appreciate you suggesting `--cwd`, didn't know about it. Could be useful in scripts and such.

Comment (sisidra) [reply docs/features/techdocs/getting-started.md:64]: Yes, my thinking here is that I tend to never leave root directory of project if I can. :)

**Review by Rugvip — COMMENTED** [review]
Thanks!

I prefer `--cwd` as well, sometimes `workspace`, but I've also seen those confuse people, which makes it less ideal to use them in them in docs.

Tbh for most of local development it makes sense to `cd` into the folder anyway, because usually you head into a plugin to work on it for a while.

Happy to continue the debate tho :grin:

**Comment by sisidra** [comment]
I have no strong opinion on this. We can wait for more opinions or close it. :) 🤷

**Comment by Rugvip** [comment]
Gonna keep it as is just to be sure we don't cause confusion, thank you for the suggestion tho!

**Pull request closed without merging**